backend/src/BackendOnlyStdLib/BackendOnlyStdLib.fsproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <ProjectReference Include="../Prelude/Prelude.fsproj" />
     <ProjectReference Include="../LibExecution/LibExecution.fsproj" />
+    <ProjectReference Include="../Parser/Parser.fsproj" />
```

Comment:
Thinking out loud, perhaps we should have a BwdServerExperiment1 where we include these?

Reply:
Hmm, what are "these" in this context?